app/models/Broadcast.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const mongoose = require('mongoose');
+
+/**
+ * Schema.
+ */
+const broadcastSchema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  type: { type: String, enum: ['signup_prompt'] },
+  campaign: Number,
+  messages: {
+    signup_prompt_declined: String,
```

Comment:
I talked to @jamjensen earlier about that, and he's okay with it. This do add another one level of complexity, but editing documents in Mongo is and advanced technique by itself.